tests/framework/jailer.py

```diff
@@ -246,7 +246,7 @@ def cleanup(self):
         if self.jailer_id:
             shutil.rmtree(self.chroot_base_with_id(), ignore_errors=True)
 
-        if self.netns:
+        if os.path.exists("/var/run/netns/{}".format(self.netns)):
```

Comment:
should we also check if `self.netns` is set ?

Reply:
self.netns is set in the constructor and it cannot be None:
`self.netns = netns if netns is not None else jailer_id`, jailer_id is the microvm_id and the microvm_id cannot be None.